pump-jack-motion-monitoring/README.md

```diff
@@ -0,0 +1,132 @@
+
+## Hardware
+
+The following hardware is required to run the Pump Jack Motion Monitor.
+
+* [Blues Notecard](https://shop.blues.io/collections/notecard)
+* [Blues Notecarrier B](https://shop.blues.io/products/carr-b)
+* A LiPo Battery of any size, like [this one](https://www.adafruit.com/product/328) from Adafruit or [from Blues](https://shop.blues.io/collections/accessories).
+
+> **NOTE**: For a detailed look at how to connect the hardware, refer to the
+[Notecard and Notecarrier B quickstart guide](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-b/).
+
+With the hardware assembled you next need to get the Notecard configured for motion tracking.
+
+## Configure Notecard
+
+This is a "host-free" project because the _Notecard itself_ can be configured to monitor the motion of whatever it is connected to using its built-in accelerometer. It may sound simple, but this is a big deal. Typically, a host microcontroller is necessary to interface with sensors and
+determine when data should be sent over the network to a cloud service. The fact that Notecard can do this without a host means big savings in terms of necessary hardware and necessary power-usage the device needs, as microcontrollers tend to be resource intensive.
+
+This section will guide you through creating a reusable Notecard configuration script which can be executed through the [Notecard CLI](https://dev.blues.io/tools-and-sdks/notecard-cli/) or through the [online webREPL](https://dev.blues.io/notecard-playground/) available on the Blues Developer Experience site.
+
+There are two main ways to program a standalone Notecard that will not be controlled by a host microcontroller:
+1. [The Notecard CLI](https://dev.blues.io/tools-and-sdks/notecard-cli/): a downloadable CLI designed for the Notecard.
+2. The [Notecard Playground](https://dev.blues.io/notecard-playground/): an in-browser terminal that emulates much of the functionality of the Notecard CLI.
+
+This guide will walk you through using the Notecard CLI, but you can also enter these configuration commands using the online REPL.
+
+1. Download and install the [Notecard CLI](https://dev.blues.io/tools-and-sdks/notecard-cli/#installation) from the Blues Developer Experience site.
+1. Plug the Notecarrier containing the Notecard into your computer with a USB-A to micro USB cable.
+1. Copy the JSON file example in this folder named [`pump-jack-notecard-configuration.json`](./configuration/pump-jack-notecard-configuration.json) to your local machine.
+1. Remove all the # comments describing what each command does, add your `product` in the `hub.set` request, and save the file. It will resemble the code snippet below when you're done.
+```json
+{"req":"env.default","name":"pump_jack_notecard_configuration_version","text":"1.0.0"}
+{"req":"hub.set","mode":"periodic","align":true,"voutbound":"usb:10;high:360;normal:720;low:1440;dead:0","vinbound":"usb:10;high:180; normal:180;low:360;dead:0"}
```

Comment:
Not a critical comment, but this might be confusing to readers. I can't think why you would ever configure outbound to be a longer duration than inbound, because the Notecard will always since at the inbound interval, where it only syncs at the outbound interval if there are unsync'd notes.  I would expect inbound to always be <= outbound.